core/dbt/node_types.py

```diff
@@ -45,25 +45,41 @@ class UnparsedNodeType(StrEnum):
     RPCCall = str(NodeType.RPCCall)
 
 
+class RunHookType(StrEnum):
+    Start = 'on-run-start'
+    End = 'on-run-end'
+
+
+class AnalysisType(StrEnum):
+    Analysis = str(NodeType.Analysis)
```

Comment:
I think you can use:

```
from hologram.helpers import StrLiteral

AnalysisType = StrLiteral(str(NodeType.Analysis))
```

to create these "literal" enums

Reply:
Good call - for a while I had them as

```
class ParsedAnalysis(ParsedNode):
   resource_type: StrLiteral(str(NodeType.Analysis))
```

And that caused annoying issues on Windows with multiprocessing (which pickles objects), so I wrote it like this. I'll make sure this works ok, but it should. I should probably add pickling unit tests for this.